### ISSUE TITLE
infowindow   can you add  autoclose option   when click map  

### ISSUE THREAD

**Issue by shanyipeng** [maptalks/maptalks.js#515]
#### Environment
* maptalks version:
* Browser (version):
* OS/Platform (version):

#### jsfiddle or codepen URL
> the example url of maptalks to attach in
>
> https://cdn.jsdelivr.net/npm/maptalks@0.30.0/dist/maptalks.js
>
> https://cdn.jsdelivr.net/npm/maptalks@0.30.0/dist/maptalks.css

**Comment by fuzhenn** [comment]
Good idea!

How about adding a new options "autoCloseOn" like below?

Let users be able to choose which events to auto close the infowindow.

```javascript
new maptalks.ui.InfoWindow({
   //...
  'autoCloseOn' : 'click contextmenu' // auto close when click or right click on map, null by default
});
```

**Comment by shanyipeng** [comment]
I like it !

**Comment by fuzhenn** [comment]
@shanyipeng thx for your proposal.

'autoCloseOn' is implemented in the commit above.

Will be released in 0.32.0
